### ISSUE TITLE
[BUG]: thumbnails won't load when requesting more pages 

### ISSUE THREAD

**Issue by dpicassom** [pystardust/ytfzf#717]
First of all, great program! I love it, very effective!

### Describe the bug
When i request more pages via `next_page_action_shortcut` new thumbnails don't ever load (entries load all right)

### To Reproduce

here's the config:

```bash
show_thumbnails=1
thumbnail_viewer="mpv"
next_page_action_shortcut="ctrl-b"
async_thumbnails=1
skip_thumb_download=0
```
### Expected behavior

for new thumbnails (of the newly added search results via  `next_page_action_shortcut`  ) to be fetched and showed async, eventually

### Screenshots
[Screencast from 2024-01-23 16-37-50.webm](https://github.com/pystardust/ytfzf/assets/59299805/cd3d7930-8db4-4d9a-b5c8-77555426f229)

Sorry for the length of the video, took me too long to request a new page.

### Information
 - OS: CachyOS
 - Terminal: kitty
 - Ytfzf version: ytfzf: 2.6.1
 - Output of `readlink $(which sh)`: `bash`
 - (if is a thumbnail issue) run `ytfzf --thumbnail-log=log.txt` and post the file:

```
[file] Cannot open file '/tmp/ytfzf-1000/some stuff-323795/thumbnails/ux-4Dwcv6CE.jpg': No such file or directory
Failed to open /tmp/ytfzf-1000/some stuff-323795/thumbnails/ux-4Dwcv6CE.jpg.
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
VO: [gpu] 480x360 yuv420p
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
[ffmpeg/video] mjpeg: EOI missing, emulating
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
 (+) Video --vid=1 (mjpeg 480x360 1.000fps)
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
V: 00:00:00 / 00:00:00
V: 00:00:00 / 00:00:00 (0%)
Exiting... (Quit)
V: 00:00:00 / 00:00:00 (0%)
```

### Additional context

I assume the first fail to load image is that of the first result, as it's loaded async, it wont be there until it's finished

Would be nice if waited to load the first one before loading results

Would be extremely nice if focus stayed on the terminal when mpv for thumbnail display is first called

the family guy image loaded because it's repeated from the first page results, the other's don't

**Comment by Euro20179** [comment]
This should now be fixed in the development branch (71b9a6d4b044390143ab12ab10a85b8f10cbec02)
> Would be extremely nice if focus stayed on the terminal when mpv for thumbnail display is first called

This is not really possible to implement in a way that would always work.
I'd recommend doing some hacky window rules in your window manager instead (i know this isn't great lol)

**Comment by dpicassom** [comment]
Thank you , good sir!
